Added GitHub Actions

daminijain23 opened the pull request:
Added -
release.yml
tox-test.yml

Deleted - 
.travis.yml

https://projects.engineering.redhat.com/browse/RHELDST-7266

rajulkumar commented on .github/workflows/tox-test.yml:
```diff
@@ -0,0 +1,50 @@
+
+on: [push, pull_request]
+
+jobs:
```
This should also include a Py27 job.

rajulkumar reviewed (changes requested):
As suggested in other review, can the commits be squashed and a descriptive commit message be added.
Also, you may update the tox.ini and retire travis specific job in this or another commit.

daminijain23 commented:
@rajulkumar 
1. Added py27 job
2. As suggested in other repo review - removed codecov token  from tox-test.ymland change pypi_password to pypi token in release.yml.
3. squashed commit.
4. Removed travis related jobs in the same commit.

rohanpm commented on .github/workflows/tox-test.yml:
```diff
@@ -0,0 +1,65 @@
+      - uses: actions/checkout@v2
+      - name: Install RPM
+        run: |
+          sudo apt-get install -y rpm
```
Minor - these could all be a single command. It would be slightly more efficient.

rajulkumar reviewed (changes requested):
[tox]
envlist = py27, py36

py36 should be changed to py38 as that's what is running in actions to avoid inconsistency. Else might introduce some confusion with tox running fine as it's using py36 but the CI running on py38 might have some errors.
